--- a/src/fl_draw.cxx	2008-09-16 05:46:48.000000000 +0000
+++ b/src/fl_draw.cxx	2026-02-14 18:00:00.000000000 +0000
@@ -32,13 +32,13 @@
 // Aligns them against the inside of the box.

-#define min(a,b) ((a)<(b)?(a):(b))
 #include <FL/fl_draw.H>
 #include <FL/Fl_Image.H>

 #include "flstring.h"
 #include <ctype.h>
 #include <math.h>

+#define min(a,b) ((a)<(b)?(a):(b))
 #define MAXBUF 1024

